### PR TITLE
Feature: 253 ubeswap

### PR DESCRIPTION
**Pull request by L03TJ3**
# Description

Adding the ubeswap component when user is connected to celo

About # (link your issue here)
#253 

# How Has This Been Tested?
locally
Please describe the tests that you ran to verify your changes.

#todo
- [ ] Move component to good-design with a wrapper (icebox)
- [x] G$ logo is not loaded (waiting for ube dev response)

### CODE REVIEW

**Comment by L03TJ3** [comment]
I think for now this basic implementation is enough there is not much configuration/styling that can be done

I suggest whenever the team open up their styling we move it to good-design

File: src/pages/gd/Swap/index.tsx [hunk 1]
```diff
@@ -320,7 +321,7 @@ const Swap = memo(() => {
                         />
                         <div className="switch">
                             {cloneElement(SwitchSVG, {
-                                onClick: () => setBuying(value => !value)
+                                onClick: () => setBuying((value) => !value),
```
Comment (johnsmith-gooddollar) [inline src/pages/gd/Swap/index.tsx:324]: no inline callbacks, define them outside jsx as useCallback

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]

**Review by johnsmith-gooddollar — APPROVED** [review]

**Comment by L03TJ3** [comment]
Current un-catched error on a user declining a connect wallet request. this originates from use-contract-kit
awaiting reply from ubeswap dev before merge

File: src/pages/gd/Swap/SwapCelo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useApplicationTheme } from 'state/application/hooks'
+import Swap from 'ubeswap-swap-dev'
+
+export const UbeSwap = (): JSX.Element => {
+    const [theme] = useApplicationTheme()
+    return (
```
Comment (johnsmith-gooddollar) [inline src/pages/gd/Swap/SwapCelo/index.tsx:7]:
```suggestion

    return (
```

**Review by johnsmith-gooddollar — APPROVED** [review]

**Pull request merged**